### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by Norikaz**
Updated some text in ReadME to personalize my portfolio.

### CODE REVIEW

**Comment by dchickabasapa** [comment]
Nitpick: It is always good to add some description to your pull request irrespective of how small the change is. Also assign a reviewer for your pull-request. We will go through these in our first team meeting.

**Review by dchickabasapa — APPROVED** [review]
This can just be:

"This repo contains Norik's protfolio and SPS projects."

**Comment by Norikaz** [comment]
Thanks for the feedback! I'll make sure to apply that in further pull requests.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Norik's protfolio and SPS projects you'll work on throughout SPS.
```
Comment (dchickabasapa) [inline README.md:3]: Nit: You can keep this string as "This repo contains Norik's protfolio and SPS projects."

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,13 +7,23 @@
...
+      <h1>Norik's Portfolio</h1>
+      <p>Hi everyone my name is Norik I am a student software engineer.</p>
+      <p>The following are some of my socials:</p>
+      <a href="https://www.linkedin.com/in/norikzhagui" target="_blank">
+        <img src="images/linkedIn.png" id="socials">
+      </a>
+      <a href="https://github.com/Norikaz" target="_blank">
+        <img src="images/github.png" id="socials">
+      </a>
+      <p>A project i have worked on is</p>
```
Comment (dchickabasapa) [inline portfolio/src/main/webapp/index.html]: Nit: replace "i" with "I"

**Review by dchickabasapa — APPROVED** [review]

**Review by dchickabasapa — APPROVED** [review]

**Review by dchickabasapa — APPROVED** [review]

**Review by dchickabasapa — APPROVED** [review]

**Pull request merged**